### PR TITLE
Open all the security holes ever for gtest as well

### PR DESCRIPTION
**Pull request by markuspf**

### CODE REVIEW

**Review by mchacki — APPROVED** [review]
Cannot say it looks good to me.

But at least it looks consistent with the other things.

I am no expert on Oskar so I cannot decide if this are all relevant places.

**Review by KVS85 — CHANGES_REQUESTED** [review]
Tags must be updated (at `helper.linux.fish` and all images recreated and updated within AWS templates).

**Comment by markuspf** [comment]
> Tags must be updated (at `helper.linux.fish` and all images recreated and updated within AWS templates).

*who* has to do this, how, and when will it happen?

**Review by KVS85 — APPROVED** [review]

File: containers/buildAlpine6.docker/Dockerfile [hunk 1]
```diff
@@ -27,13 +27,6 @@ COPY ./scripts /scripts
...
 
-RUN git config --global --add safe.directory /work/ArangoDB
-RUN git config --global --add safe.directory /work/ArangoDB/enterprise
-RUN git config --global --add safe.directory /work/ArangoDB/docs
-
-RUN git config --global --add safe.directory /work/ArangoDB/3rdParty/fmt
-RUN git config --global --add safe.directory /work/ArangoDB/3rdParty/iresearch
-RUN git config --global --add safe.directory /work/ArangoDB/3rdParty/rocksdb
-RUN git config --global --add safe.directory /work/ArangoDB/3rdParty/velocypack
+RUN git config --global --add safe.directory '*'
```
Comment (markuspf) [inline containers/buildAlpine6.docker/Dockerfile:30]: lol.

**Comment by KVS85** [comment]
- [ ] https://jenkins01.arangodb.biz/job/arangodb-matrix-pr/24835/

**Pull request merged**